src/network/server_connection.rs

```diff
@@ -87,18 +88,26 @@ impl ServerConnection {
         route: impl AsRef<str>,
         query: Query<'_, Q>,
         body: Body<'_, B>,
+        header: Option<HeaderMap>,
```

Comment:
this should not be required, as reqwest adds the json header automatically

Reply:
No, it doesn't. It doesn't work without it, but works with it.

Reply:
Theres literally code in reqwest that adds the header